### DISCUSSION TITLE
[Office Hours] PRISMA: User Authentication with React and Firebase

### DISCUSSION THREAD

**Discussion by EvelynBunnyDev** [CS342/.github#76] [Office Hours]
### Related Discussion

I am implementing user authentication with a web chat interface hosted in the iOS environment. I have a question regarding frontend web dev: I created a login page for authentication in web, and a hard-coded user id and password for the sake of simplicity. However, clicking the login button does not bring the user to the next stage, even though I've written the hook and listener correctly.

### Reproduction

I will share my code during the office hour.

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
@EvelynBunnyDev Thank you for asking a question!

Did you manage to resolve this issue in the office hours? If so, can you share what the issue was and how you resolved if for students that might struggle with the same thing in the future?

It would be great to see the implementation by the team meeting on Monday merged into the repo to ensure that we can focus on the client-focused authentication. Please be sure that you use the office ours and react tutorials & documentation to make progress on the task 🚀
